DFC.ServiceTaxonomy.Slack/SlackMessagePublisher.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+
+namespace DFC.ServiceTaxonomy.Slack
+{
+    public class SlackMessagePublisher : ISlackMessagePublisher
+    {
+        private readonly SlackMessagePublishingConfiguration _config;
+        private readonly HttpClient _client;
+
+        public SlackMessagePublisher(IOptions<SlackMessagePublishingConfiguration> config, HttpClient client)
```

Comment:
Might be an idea to use `IOptionsMonitor` (and not cache the value), as it might be useful to disable/enable the slack messages on the fly.